packages/svelte/messages/compile-warnings/a11y.md

```diff
@@ -1,167 +1,167 @@
-## a11y_aria_attributes
+## a11y_accesskey
 
-> <%name%> should not have aria-* attributes
+> Avoid using accesskey
```

Comment:
I don't know if you want to make this part of this PR, but we have https://svelte.dev/docs/accessibility-warnings with details on many of those warnings, which you could copy-paste here (and let them get autogenerated later on)